addons/hr_expense/models/hr_expense.py

```diff
@@ -613,8 +613,15 @@ def get_expense_dashboard(self):
         expenses = self._read_group(
             [
                 ('employee_id', 'in', self.env.user.employee_ids.ids),
-                ('payment_mode', '=', 'own_account'),
-                ('state', 'in', ('draft', 'reported', 'submitted', 'approved'))
+                # For states draft, reported and submitted: payment_mode should be in (own_account, company_account),
```

Comment:
I'm not sure the comment is necessary since it only tells what is written. I don't mind if you keep it tho.

Reply:
I'd say that the comment is quite useless in that form, I'd rather have something like

```python
# Counting the expenses to display in the dashboard:
# - To submit: contains the expenses paid either by the employee or by the company, and that are draft or  reported
# - Under validation: contains expenses paid by the employee or paid by the company, and that have been submitted but still need to be approved/refused
# - To be reimbursed: contains ONLY expenses paid by the employee that are approved, the payment has not yet been made
```